app/frontend/js/map.js

```diff
@@ -110,12 +109,25 @@ const convertToParcelSheetId = (parcelId) => {
 
 const addCheckboxEventListener = (checkbox, selectfeatures, parcelSource) => {
   checkbox.addEventListener('change', (e) => {
-    const parcelId = convertToParcelSheetId(checkbox.id)
+    //console.info('Parcel parcelSource', checkbox.id)
     const parcelFeatures = parcelSource.getFeatures()
-    for (const feature of parcelFeatures) {
-      if (feature.get('parcel_id') === parcelId[1] && feature.get('sheet_id') === parcelId[0]) {
+    if(checkbox.id === "SelectAll") {
+      for (const feature of parcelFeatures) {
```

Comment:
Could this be in its own function. So split check all and check individual into their own functions